### PR TITLE
Fix Neos 7.0 compatibility

### PR DESCRIPTION
**Pull request by bwaidelich**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
         "neos/neos-ui": "@dev",
         "neos/seo": "~3.0",
         "neos/redirecthandler-neosadapter": "~4.0",
```
Comment (bwaidelich) [inline composer.json]:
```suggestion
        "neos/redirecthandler-neosadapter": "~4.3",
```

**Review by jonnitto — APPROVED** [review]

**Review by markusguenther — APPROVED** [review]
Looks good by reading

**Comment by jonnitto** [comment]
`neos/redirecthandler-ui` could also be lifted to 2.4

File: composer.json [hunk 2]
```diff
@@ -22,8 +22,8 @@
         "neos/demo": "@dev",
         "neos/neos-ui": "@dev",
         "neos/seo": "~3.0",
-        "neos/redirecthandler-neosadapter": "~4.0",
-        "neos/redirecthandler-databasestorage": "~4.0",
+        "neos/redirecthandler-neosadapter": "~4.3",
+        "neos/redirecthandler-databasestorage": "~5.0",
         "neos/redirecthandler-ui": "~2.0",
```
Comment (jonnitto) [inline composer.json]:
```suggestion
        "neos/redirecthandler-ui": "~2.4",
```

**Pull request merged**